### PR TITLE
Dev/rabbitmq and localization

### PR DESCRIPTION
**Pull request by deadshot465**
A branch called `rabbitmq_and_localization` but primarily consists of NinetyNine codes.

### CODE REVIEW

File: BurstBotNET/Config/config.yaml.example [hunk 1]
```diff
@@ -9,4 +9,7 @@ recreateGuilds: "false"
 serverEndpoint: ""
 socketEndpoint: "localhost"
 socketPort: 8080
-timeout: 300
\ No newline at end of file
+timeout: 300
+rabbitMqEndpoint: ""
+rabbitMqUsername: ""
+rabbitMqPassword: ""
```
Comment (MorphTuple) [inline BurstBotNET/Config/config.yaml.example]: It can be grouped

```yaml
rabbit:
  endpoint: ""
  username: ""
  password: ""
```

**Review by MorphTuple — CHANGES_REQUESTED** [review]
Everything else LGTM, for the RabbitMQ usage I think it makes sense to use a tiny client wrapped in a service class like this than using some sort of framework like MassTransit.

File: BurstBotNET/Config/config.yaml.example (hunk 1 above)
Comment (deadshot465) [reply BurstBotNET/Config/config.yaml.example]: Oh that's indeed nicer

**Review by MorphTuple — APPROVED** [review]
LGTM

**Pull request merged**